torch/_inductor/codegen/triton.py

```diff
@@ -2255,16 +2255,38 @@ def can_fuse(self, node1, node2):
         _, (numel2, rnumel2) = node2.group
 
         if node1.is_reduction() and node2.is_reduction():
-            return numel1 == numel2 and rnumel1 == rnumel2
+            reduction_can_fuse = numel1 == numel2 and rnumel1 == rnumel2
```

Comment:
This will potentailly add a lot of log spam, I'd say create a logging artifact called "inductor_fusions" or something like that, and set `off_by_default` to True. 

See the registrations here for examples: 
https://github.com/pytorch/pytorch/blob/efb73fe8e4413a0d6db078e85c7ed7c91f05ca5d/torch/_logging/_registrations.py#L4

Reply:
yes, for large models, it could be a lot of spam.

Reply:
hid behind `TORCH_LOGS="+schedule"`

Reply:
Typically with artifacts, we don't support levels (ie +) it's always debug, could you create a separate artifact rather than grouping with the scheduler? I think there's other scheduler logs that would be hidden by adding the fusions to this log.

Reply:
Ok, changed to `fusion`